lib/clerk/rack_middleware_v2.rb

```diff
@@ -239,5 +250,14 @@ def verify_token(token)
     def sdk
       Clerk::SDK.new
     end
+
+    def interstitial_headers(reason: nil, message: nil, status: nil)
+      {
+        "Content-Type" => "text/html",
+        "x-clerk-auth-reason" => reason,
+        "x-clerk-auth-message" => message,
+        "x-clerk-auth-status" => status,
```

Comment:
Any reason we're not using the canonical form in these header names? Are these converted by Rack I assume?

Reply:
There was no reason. I updated the code to use the canonical form.